### ISSUE TITLE
USWDS-Site: Draft content for button group accessibility checklist page

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds-site#2986]
## Summary
Draft the website content for the button group accessibility checklist items in the [Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing) (Google Doc :lock:)

## Tasks
- [x] Draft checklist content and send for review with A11Y
- [x] Final edits to checklist content

**Comment by finekatie** [comment]
Hi @alex-hull , button group gherkins are ready for review!

**Comment by alex-hull** [comment]
@amyleadem @cathybaptista @mlloydbixal @RachelCorsino I've reviewed this, and should be ready for y'all. Marking as done.
